examples/share/.lando.yml

```diff
@@ -0,0 +1,81 @@
+# share command tests
+name: share 
+
+# Give me http://lamp.lndo.site and https://lamp.lndo.site
+proxy:
+  appserver:
+    - share.lndo.site
+
+# Spin up services to run a basic LAMP stack
+services:
+
+  # Create a php 5.3 instance served by the latest apache with https turned on
+  appserver:
+
+    # PHP version 5.3
+    type: php:5.3
+
+    # Serve php by either apache or nginx. You can put in any `service:version`
+    # string that is supported by Lando's nginx and apache services.
+    via: apache
+
+    # Optionally serve php over https
+    ssl: true
+
+    # Optionally specify a webroot if it is a subdirectory of the appRoot
+    webroot: www
+
+    # Optionally customize the configuration for php and its underlying webserver
+    # These paths are both relative to (and must reside within) the app root directory
+    #
+    # For some examples that use this fairly well please check out
+    # https://github.com/lando/lando/tree/master/examples
+    # config:
+      # Use a custom apache config file
+      # server: config/apache/lamp.conf
+
+      # Use a custom php.ini
+      # This **should** load last which means you should be able to override
+      # any previously set php settings.
+      # conf: config/php/php.ini
+
+    # Optionally activate xdebug
+    #
+    # If you are having trouble getting xdebug to work please see:
+    # https://docs.devwithlando.io/services/php.html#using-xdebug
+    xdebug: true
+
+    # Optionally install the following composer dependencies globally
+    # These follow the normal composer.json form
+    # Note that many composer dependencies should not be installed globally but
+    # on a per-project basis
+    # composer:
+    #   drush/drush: "~8"
+
+  appserver:
+    overrides:
+      services:
+        ports:
+          - '8081:80'
+
+  # My database service
+  database:
```

Comment:
we can probably make this test faster by not adding extra services. A bare minimum apache/nginx, or even node service that can boot fast and just shoot back a response would speed up the tests and be lower bandwidth

Reply:
👍 good suggestion

Reply:
The problem i'm running into here is that the base services `apache` and `nginx` do not have curl installed on the base image.  This test depends on making a curl request.

The simplest thing I've found so far that has curl is `lamp` and `lemp` recipes.

I could add this test to one of those recipes ( that is what I initially did), but it also requires specifying a fixed port and I did not necessarily want to introduce that into those existing tests ... although we could.

Reply:
@serundeputy ahhhh right, that makes sense to me. i'd just roll with the `lamp` stack then since it's one fewer containers than `lemp`